app/src/main/java/com/conupods/OutdoorMaps/MapInitializer.java

```diff
@@ -115,13 +116,21 @@ public void initializeFloorButtons(View floorButtons) {
 
     // The two campus swap buttons
     public void initializeToggleButtons(Button sgwButton, Button loyButton) {
+        mSgwButton = sgwButton;
```

Comment:
if u turn them into member variables, they need to be initialized in the constructor

Reply:
Changed the constructor and its uses everywhere